### ISSUE TITLE
Statusbar percentage crashes on python 2.6.6

### ISSUE THREAD

**Issue by p-friedrich** [ranger/ranger#679]
The calculation of the position marker (TOP, BOT, 34%) in the bottom right corner of the statusbar crashes on python 2.6.6 with the following traceback:

```
ranger version: 1.7.2, executed with python 2.6.6
Locale: de_DE.UTF8
Current file: -----------------------------
Traceback (most recent call last):
  File "/home/-------/.local/lib/python2.6/site-packages/ranger/core/main.py", line 145, in main
    fm.loop()
  File "/home/-------/.local/lib/python2.6/site-packages/ranger/core/fm.py", line 351, in loop
    ui.redraw()
  File "/home/-------/.local/lib/python2.6/site-packages/ranger/gui/ui.py", line 295, in redraw
    self.draw()
  File "/home/-------/.local/lib/python2.6/site-packages/ranger/gui/ui.py", line 322, in draw
    DisplayableContainer.draw(self)
  File "/home/-------/.local/lib/python2.6/site-packages/ranger/gui/displayable.py", line 248, in draw
    displayable.draw()
  File "/home/-------/.local/lib/python2.6/site-packages/ranger/gui/widgets/statusbar.py", line 100, in draw
    self._calc_bar()
  File "/home/-------/.local/lib/python2.6/site-packages/ranger/gui/widgets/statusbar.py", line 106, in _calc_bar
    self._get_right_part(bar)
  File "/home/-------/.local/lib/python2.6/site-packages/ranger/gui/widgets/statusbar.py", line 293, in _get_right_part
    right.add('{:0.0%}'.format(float(pos) / max_pos),
ValueError: zero length field name in format
```

I suspect this is a compatibility issue with the rather old python 2.6. Despite being old, it is listed as supported python version, so this should either be fixed, or support for 2.6 dropped.

Greetings,
p-friedrich

**Comment by richboss** [comment]
I can confirm this bug. But it can easily be fixed by adding a single character, "0" (Zero) in statusbar.py in line 293, like this:

Before
`right.add('{:0.0%}'.format(float(pos) / max_pos),`

After
`right.add('{0:0.0%}'.format(float(pos) / max_pos),`

Python 2.6 requires a field name here, but in Python 2.7 it is optional. Quoted from the Pythong string docs:

> Changed in version 2.7: The positional argument specifiers can be omitted, so '{} {}' is equivalent to '{0} {1}'.

**Comment by lkraav** [comment]
I can confirm this is a real thing. `python-2.6` is still a thing on a bunch of CentOS situations.

**Comment by nfnty** [comment]
Fixed by #694.
